src/components/SideBar/style.js

```diff
@@ -0,0 +1,154 @@
+import styled from "styled-components";
+import { NavLink as Link } from "react-router-dom";
+import { BarsOutlined } from "@ant-design/icons";
+import { Input } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+const { Search } = Input;
+
+export const StyledSidebar = styled.div`
+  background: #363740;
+  color: #a4a6b3;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+  min-height: 100%;
+  height: 100vh;
+`;
+
+export const StyledSideBarContainer = styled.div`
+  @media (max-width: 800px) {
+    transition: width 0.3s ease;
+    top: 0;
+    left: 0;
+    position: absolute;
+    width: ${(props) => (props.isOpen ? "255px" : "80px")};
+  }
+`;
+export const StyledText = styled.div`
+  @media (max-width: 800px) {
+    opacity: ${(props) => (props.isOpen ? "1" : "0")};
+    transition: opacity 0.3s ease;
+    visibility: ${(props) => (props.isOpen ? "visible" : "hidden")};
+  }
+`;
+
+export const StyledLink = styled(Link)`
+  color: #a4a6b3;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+  text-decoration: none;
+  display: flex;
+  padding: 20px 30px;
+  align-items: center;
+  transition: all 0.2s;
+  opacity: 0.4;
+  position: relative;
+  &.active {
```

Comment:
For better readability I would suggest to add empty lines between each block of styles - root styles, sub styles, sub-sub styles and so on